Makefile

```diff
@@ -172,8 +172,8 @@ proto: buildkitd
 .PHONY: talos-gce
 talos-gce:
 	@docker run --rm -v /dev:/dev -v $(PWD)/build:/out --privileged $(DOCKER_ARGS) autonomy/installer:$(TAG) install -n disk -r -p googlecloud -u none
-	@tar -C $(PWD)/build -Sczf $(PWD)/build/$@.tar.gz disk.raw
```

Comment:
I think we want to keep the `-S` so we aren't tarring up a bunch of zeros. We will need to update the build-container with an `apk add tar`, the busy box implementation doesn't support `-S` it seems.

Reply:
See comment above. For some reason we don't have the -S or --sparse flag in the tar binary that the drone runner has installed.

Reply:
its busy box implementation :/